src/controllers/distance.js

```diff
@@ -0,0 +1,20 @@
+export function findDistanceToTrail(trail, coordinates) {
+  const distance = latLonDistance(trail.latitude, trail.longitude, coordinates);
+  return distance;
+}
+
+function latLonDistance(trailLat, trailLon, coordinate) {
```

Comment:
IMO this should have taken two coordinates as parameters. As far as the math goes I'm pretty lost, maybe link an article or something in a comment?

Reply:
Maybe use [Polar coordinate flat-Earth formula](https://en.wikipedia.org/wiki/Geographical_distance) with a separate function to convert to colatitude in radians?

Reply:
Yeah just took it straight from here - https://www.geodatasource.com/developers/javascript

Reply:
Also we definetly can refactor into both being objects and then just referencing .lat / .lon